test/geocoder_test.exs

```diff
@@ -8,7 +8,7 @@ defmodule GeocoderTest do
     {:ok, _} = Supervisor.restart_child(Geocoder.Supervisor, Geocoder.Store)
 
     # OpenStreetData is rate-limited at 1rps. Let's ensure our tests don't break that rate limit.
-    Process.sleep(1_000)
+    System.get_env("PROVIDER", "fake") != "fake" && Process.sleep(1_000)
```

Comment:
Let's change this to something like `System.get_env("PROVIDER") == "openstreetmaps" && Process.sleep(1_000)`

Reply:
My thought was if specs were running against google maps or some other provider (basically anything but `fake`), we would want the sleep to still be there so as to not hit an API rate limit. But if the desire is to exclusively run this file against `openstreetmaps` then I can make that change.

Reply:
That's a good point! lgtm